### ISSUE TITLE
Unable to access font properties for a paragraph if document is in only read mode (Viewing).

### ISSUE THREAD

**Issue by neer3** [OfficeDev/office-js#5039]
#### Note: The issue is happening when I am setting the document protection as `Always Open as Read Only`

So, I am facing an issue where I am trying to read all of the paragraphs present in the document, and along with that I am reading the font styling for them i.e. whether they are bold, italic, or underlined. 

## Environment

* Platform: [PC desktop, Mac, Office on the web]
* Host: Word
* Office.js version number: 1.1
* Word application version: 2410
* Operating System: Windows 10
* Browser: Chrome
 
## Expected behavior
The behavior should be that the font properties should be able to load properly without any error.

## Current behavior
Currently, I am getting a General Exception thrown stating.
`GeneralException: Cannot read properties of null (reading 'WZc')`

The above error would be thrown whenever I trigger the load and waiting on the context to sync.
Sample code ->
```
    const paragraphs = context.document.body.paragraphs;
    paragraphs.load({ $all: true });

    await context.sync();
 
    for (let i = 0; i < paragraphs.items.length; i++) {
       paragraphs.items[i].font.load({ $all: true });
    }

    await context.sync();

    // Document will always have more than 5 paragraphs.
    console.log(paragraphs.items[0].font.bold);
```

The point where the error is thrown by the Office.js

```
get bold() {
    if (null !== this.hg) {
        var c = tm.pr(this.hg, this.Qf);
        if (c.hasProperty(Sc.a.bold))
            return c.getValue(Sc.a.bold)
    } else
        return c = this.d_(),
        // here we are getting c as NULL
        c.WZc ? null : 1 === c.bold
}
```

## Steps to reproduce

1. Create a sample doc with at least 5 paragraphs.
2. Set the protect document to always open read-only, via the info options on the Word Application.
3. Open the same document on the browser and try to load the font via the add-in.

## Context

This issue is affecting to figure out the font styling for each paragraph.

**Comment by RuizhiSunMS** [comment]
Thank you for reporting this issue, I can repro it. It has been put on our backlog #9506484 , the team will investigate and we will reply to you as soon as there is any progress. Thank you for your patience.
Btw, would you please confirm that this case only happens on web?

**Comment by neer3** [comment]
Hi @RuizhiSunMS, thanks for the reply.

This is happening on the Word standalone application on Windows also for one of the users but I am not able to reproduce it on the standalone Word application.

**Comment by xiruatms** [comment]
@neer3 , is this issue still reproducable please?

**Comment by neer3** [comment]
Hi @xiruatms , I no longer have access to the environment where this issue was encountered and cannot reproduce it at this time. Feel free to close this issue. If others encounter the same problem, they can open a new issue with current details.
